conda/recipes/legate-dataframe/meta.yaml

```diff
@@ -70,32 +69,60 @@ requirements:
     - libcudf {{ rapids_version }}
     - librmm {{ rapids_version }}
     - pylibcudf {{ rapids_version }}
-  run:
-    - python
-    # cuda-version is used to constrain __cuda
-    - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
-    - cuda-cudart
-    - __cuda
-    # Relying on run_exports from legate to pin an appropriate range of versions.
-    #
-    # legate uses {{ pin_subpackage(name, min_pin="x.x.x", max_pin="x.x.x") }}, which means
-    # that if legate-dataframe builds against, say, legate=24.09.0 and then a legate=24.09.1 is
-    # released, a new legate-dataframe would be required.
-    - legate
-    - numpy >=1.23,<3.0.0a0
-    - rmm {{ rapids_version }}
-    - cudf {{ rapids_version }}
 
-test:
-  # import tests at build time do not work for the CUDA packages,
-  # because builds happen on machines without a GPU.  So no tests.
-  commands:
-    - pip show legate-dataframe
+
+outputs:
+  - name: legate-dataframe
+    version: {{ version }}
+    string: "cuda{{ cuda_major }}_py{{ py_version }}_{{ PKG_BUILDNUM }}"
+    script: install_legate-dataframe.sh
+    requirements:
+      build:
+        - cmake {{ cmake_version }}
+      host:
+        - cuda-version ={{ cuda_version }}
+        - python
```

Comment:
Without Python here, I think I got a Python 3.13 rather than the same Python (at in my local build).

Reply:
`cuda-version` in `host` will help ensure you get the expected version of `nvcc` in the `host` environment, I think. This is good to keep.